ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -78,6 +78,61 @@ objects:
 
     containers:
 
+    - name: lease
+      image: ci/boskoscli:latest
+      terminationMessagePolicy: FallbackToLogsOnError
+      resources:
+        requests:
+          cpu: 10m
+          memory: 10Mi
+        limits:
+          memory: 200Mi
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp/shared
+      - name: cluster-profile
+        mountPath: /tmp/cluster
+      - name: artifacts
+        mountPath: /tmp/artifacts
+      env:
+      - name: CLUSTER_TYPE
+        value: ${CLUSTER_TYPE}
+      - name: CLUSTER_NAME
+        value: ${NAMESPACE}-${JOB_NAME_HASH}
+      command:
+      - /bin/bash
+      - -c
+      - |
+        #!/bin/bash
+        set -euo pipefail
+
+        trap 'kill $(jobs -p); exit 0' TERM
+
+        if [[ "${CLUSTER_TYPE:-}" != "aws" ]]; then
+          echo "[WARNING] No resource leasing is defined for \"${CLUSTER_TYPE}\" clusters"
+          exit 0
+        fi
+
+        resource="$( boskosctl --server-url http://boskos --owner-name "${CLUSTER_NAME}" acquire --type aws-quota-slice --state clean --target-state leased --timeout 30m )"
```

Comment:
Is `http://<service-name>` reachable this way in a different namespace? Or do I need a more fully qualified domain name? I don't know what I am doing

Reply:
From [these][1] [docs][2], I'd expect you need at least `${service}.${namespace}`.  So with a `boskos` service in a `boscos` namespace, `http://boscos.boscos` should work, and `http://boscos.boscos.svc.cluster.local.` would be more obvious.

[1]: https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#what-things-get-dns-names
[2]: https://github.com/kubernetes/dns/blob/master/docs/specification.md#23---records-for-a-service-with-clusterip

Reply:
Namespace added